deepchem/data/data_loader.py

```diff
@@ -186,7 +187,20 @@ def __init__(self,
     self.log_every_n = log_every_n
 
   def featurize(self, input_files, data_dir=None, shard_size=8192):
-    """Featurize provided files and write to specified location."""
+    """Featurize provided files and write to specified location.
+    
+    For large datasets, automatically shards into smaller chunks
+    for convenience.
+
+    Parameters
+    ----------
+    input_files: list
```

Comment:
files or strings?

Reply:
its strings

Reply:
Good point. Fixed description to state "filenames"